modules/tinymce/src/core/main/ts/fmt/CaretFormat.ts

```diff
@@ -100,18 +100,25 @@ const removeCaretContainerNode = (editor: Editor, node: Node, moveCaret: boolean
   } else {
     const rng = selection.getRng();
     const block = dom.getParent(node, dom.isBlock);
+
+    // Store the current selection offsets
+    const startContainer = rng.startContainer;
+    const startOffset = rng.startOffset;
+    const endContainer = rng.endContainer;
+    const endOffset = rng.endOffset;
```

Comment:
This is a big reason TBIO generally relies on a data structure with these four range details, not the range itself. Perhaps we should introduce that to TinyMCE (not now, later) as there are likely other places it will help 🤔